2022-04-10 17:58:18.215 | INFO     | db:__init__:23 - Datebase Class Created With <module 'mysql' from '/usr/local/lib/python3.9/dist-packages/mysql/__init__.py'>
2022-04-10 17:58:20.229 | DEBUG    | main:startScrapperSchdular:37 - startScrapper
2022-04-10 17:58:20.273 | ERROR    | main:Error:55 - 
 HTTPException Error : 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. 

2022-04-10 17:58:20.273 | ERROR    | main:error:67 - 
Error(name, code)->-----------name Not Found   code 404

